src/test/scala/chiseltest/formal/MagicPacketTrackerTests.scala

```diff
@@ -0,0 +1,383 @@
+package chiseltest.formal
+
+import chisel3._
+import chisel3.util._
+import chiseltest._
+import firrtl.AnnotationSeq
+import org.scalatest.flatspec.AnyFlatSpec
+
+class MagicPacketTrackerTests extends AnyFlatSpec with ChiselScalatestTester with Formal with FormalBackendOption {
+  behavior of "MagicPacketTracker"
+
+  private def defaultOptions: AnnotationSeq = Seq(BoundedCheck(7), DefaultBackend)
```

Comment:
How exactly does this work? What is DefaultBackend and how is it materialized when the test closures are run?